diff --git a/conf/defaults.ini b/conf/defaults.ini
index 258a019..6a7bd6b 100644
--- a/conf/defaults.ini
+++ b/conf/defaults.ini
@@ -10,11 +10,11 @@ app_mode = production
 [paths]
 # Path to where grafana can store temp files, sessions, and the sqlite3 db (if that is used)
 #
-data = data
+data = /var/lib/grafana
 #
 # Directory where grafana can store logs
 #
-logs = data/log
+logs = /var/log/grafana
 #
 # Directory where grafana will automatically scan and look for plugins
 #
@@ -45,7 +45,7 @@ root_url = %(protocol)s://%(domain)s:%(http_port)s/
 router_logging = false

 # the path relative working path
-static_root_path = public
+static_root_path = /usr/share/grafana/public

 # enable gzip
 enable_gzip = false
@@ -73,7 +73,7 @@ client_cert_path =
 server_cert_name =

 # For "sqlite3" only, path relative to data_path setting
-path = grafana.db
+path = /var/lib/grafana/grafana.db

 #################################### Session ####################################
 [session]
